src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2623,8 +2623,8 @@ class EditorInlayHints extends BaseEditorOption<EditorOption.inlayHints, IEditor
 					enum: ['on', 'onUnlessPressed', 'offUnlessPressed', 'off'],
 					markdownEnumDescriptions: [
 						nls.localize('editor.inlayHints.on', "Inlay hints are enabled"),
-						nls.localize('editor.inlayHints.onUnlessPressed', "Inlay hints are showing by default and hide when holding `Ctrl+Alt`"),
-						nls.localize('editor.inlayHints.offUnlessPressed', "Inlay hints are hidden by default and show when holding `Ctrl+Alt`"),
+						nls.localize('editor.inlayHints.onUnlessPressed', "Inlay hints are showing by default and hide when holding `Ctrl+Alt` on Windows and Linux and to `Ctrl+Option` on macOS."),
```

Comment:
You can use variables within these `nls.localize` functions and dynamically change the description based on operating system. I don't love spelling out Windows info to Mac users and vice versa.

Code pointer:
https://github.com/microsoft/vscode/blob/f51200c983f90a1ed46be19c65b9ca355d8a0b0f/src/vs/platform/undoRedo/common/undoRedoService.ts#L1188-L1191

Reply:
> You can use variables within these `nls.localize` functions and dynamically change the description based on operating system. I don't love spelling out Windows info to Mac users and vice versa.
> 
> Code pointer:
> 
> https://github.com/microsoft/vscode/blob/f51200c983f90a1ed46be19c65b9ca355d8a0b0f/src/vs/platform/undoRedo/common/undoRedoService.ts#L1188-L1191

Yes, indeed, I changed to use ‘platform’ variable, but I did not consider the iOS platform